Formula/c2rust.rb

```diff
@@ -0,0 +1,31 @@
+class C2rust < Formula
+  desc "Migrate C code to Rust"
+  homepage "https://c2rust.com/"
+  url "https://github.com/immunant/c2rust/archive/refs/tags/v0.16.0.tar.gz"
+  sha256 "9ed1720672afb503db91b30cec1dedcf878841f57eaea4c7046839890990d8cd"
+  license "BSD-3-Clause"
+
+  depends_on "cmake" => [:build, :test]
+  depends_on "rust" => :build
+  depends_on "llvm"
+
+  on_linux do
+    depends_on "gcc"
+  end
+
+  fails_with gcc: "5"
+
+  def install
+    cd "c2rust" do
+      system "cargo", "install", *std_cargo_args
+    end
```

Comment:
Any reason we're not doing this?
```suggestion
    system "cargo", "install", *std_cargo_args(path: "c2rust")
```

Reply:
No, updated